src/main/resources/application-prod.yml

```diff
@@ -26,6 +26,6 @@ faf-client:
   status-page-url: https://statuspage.faforever.com
 
   forged-alliance:
-    exe-url: https://content.faforever.com/faf/updaterNew/updates_faf_files/ForgedAlliance.exe
+    exe-url: https://content.faforever.com/updaterNew/updates_faf_files/ForgedAlliance.exe
```

Comment:
like that @Brutus5000 ?

Reply:
does not work so i guess not

Reply:
think it was only /faf/vault

Reply:
just /faf is fine

Reply:
Indeed /faf is fine. It was just that the /faf/vault/<folder> was actually a symlink on the server to ../../<folder> so I removed it